pkg/applicationmanager/controllers/applicationbackup.go

```diff
@@ -1260,66 +1262,74 @@ func (a *ApplicationBackupController) backupResources(
 	return nil
 }
 
-func (a *ApplicationBackupController) deleteBackup(backup *stork_api.ApplicationBackup) error {
+func (a *ApplicationBackupController) deleteBackup(backup *stork_api.ApplicationBackup) (bool, error) {
 	// Only delete the backup from the backupLocation if the ReclaimPolicy is
 	// set to Delete or if it is not successful
 	if backup.Spec.ReclaimPolicy != stork_api.ApplicationBackupReclaimPolicyDelete &&
 		backup.Status.Status == stork_api.ApplicationBackupStatusSuccessful {
-		return nil
+		return false, nil
```

Comment:
In this case  we might need to return true rt? Otherwise it my loop.

Reply:
Right, fixed it.